docs/user/alerting/rule-types/index-threshold.asciidoc

```diff
@@ -7,7 +7,7 @@ The index threshold rule type runs an {es} query. It aggregates field values fro
 [float]
 ==== Create the rule
 
-Fill in the <<defining-rules-general-details, rule details>>, then select *Index Threshold*.
+Fill in the name and optional tags, then select *Index Threshold*.
```

Comment:
We still have a check interval that the user can change, it just has moved down the flyout to be closer to the actions. Should we mention something about it in the docs?

Reply:
Yup, I'll be updating the screenshots and text for each rule type in a separate PR since there are a lot of them (listed in https://github.com/elastic/kibana/issues/150655)!  I just needed to remove this link for the build to pass.